### PR TITLE
Week 2 Sever Development

### PR DESCRIPTION
**Pull request by d-etienne**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -15,18 +15,45 @@
...
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
+      String comment = getComment(request);
+      comments.add(comment);
+      response.sendRedirect("/index.html");
+    }
```
Comment (emto) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:70]: fix indent

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -15,18 +15,45 @@
...
+
+
+  private String convertToJson(ArrayList comments){
+    Gson gson = new Gson();
+    String json = gson.toJson(comments);
+    return json;
+  }
+
+    private String getComment(HttpServletRequest request) {
+        String comment = request.getParameter("comment-box");
```
Comment (emto) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: You can do this in 1 line. 

return request.getParameter("comment-box")

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -19,10 +19,48 @@ function addFunFacts() {
...
   const funFact = funFacts[Math.floor(Math.random() * funFacts.length)];
 
   // Add it to the page.
   const factContainer = document.getElementById('funFact-container');
   factContainer.innerText = funFact;
 }
+
+/*
+
+function getComments() {
```
Comment (emto) [inline portfolio/src/main/webapp/script.js]: remember to delete this

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -14,15 +14,27 @@ body {
   background: black;
   margin-left: auto;
   margin-right:auto;
+  /*margin-left: auto;
```
Comment (emto) [inline portfolio/src/main/webapp/style.css]: delete your commented out code so its cleaner :)

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -14,15 +14,27 @@ body {
...
 }
 .center{
     text-align: center;
 }
 .card {
   background-color: white;
+  float: center; 
+  width: 70%;
   padding: 20px;
-  margin-top: 20px;
```
Comment (emto) [inline portfolio/src/main/webapp/style.css:25]: delete

**Review by emto — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (d-etienne) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:70]: Ive made this change :)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (d-etienne) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Ive made this change :)

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (d-etienne) [reply portfolio/src/main/webapp/script.js]: Ive made this change :)

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (d-etienne) [reply portfolio/src/main/webapp/style.css]: Ive made this change :)

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (d-etienne) [reply portfolio/src/main/webapp/style.css:25]: Ive made this change :)

**Review by emto — APPROVED** [review]

**Pull request merged**